core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -660,4 +690,44 @@ public RestConfig(ConfigDef definition) {
   public Time getTime() {
     return defaultTime;
   }
+
+  public static void validateHttpResponseHeaderConfig(String config) {
+    try {
+      // validate format
+      String[] configTokens = config.trim().split("\\s+", 2);
+      if (configTokens.length != 2) {
+        throw new ConfigException(String.format("Invalid format of header config for %s "
+                + "The format must be in [ation] [header name]:[header value]", config));
+      }
+
+      // validate action
+      String method = configTokens[0].trim();
+      validateHeaderConfigAction(method.toLowerCase());
+
+      // validate header name and header value pair
+      String header = configTokens[1];
+      String[] headerTokens = header.trim().split(":");
+      if (headerTokens.length > 2) {
+        throw new ConfigException(String.format("Invalid format of header name and header value"
+                + "in %s . The format must be in [header name]:[header value]", header));
+      }
+
+      // validate header name
+      String headerName = headerTokens[0].trim();
+      if (headerName.contains(" ")) {
+        throw new ConfigException(String.format("Invalid header name %s ."
```

Comment:
I think you need to put all of the supplied values in these error messages in quotation marks. If there were a trailing space in the header name, this would be confusing. It would look like 
```
Invalid header name X-bad-header  .The [header name] cannot contain whitespaces
```
Note that the spacing around the period is wrong as well. There should be a space after the period, not before.

Quotation marks would make it clearer
```
Invalid header name: "X-bad-header ". The [header name] cannot contain whitespaces
```